docs/provisioner/08-02-provisioning-gardener.md

```diff
@@ -105,6 +105,7 @@ This tutorial shows how to provision clusters with Kyma Runtimes on Google Cloud
                 machineType: "n1-standard-4"
                 region: "europe-west4"
                 provider: "gcp"
+                purpose: "testing" # possible values: "development", "evaluation", "production", "testing"; default value: "development"
```

Comment:
There is no default value. If you make a call without this field, the cluster will be created with an `evaluation` purpose